src/test/java/org/prebid/server/ApplicationTest.java

```diff
@@ -695,6 +702,21 @@ public void versionHandlerShouldRespondWithCommitRevision() {
                 .statusCode(200);
     }
 
+    @Test
+    public void currencyRatesHandlerShouldRespondWithLastUpdateDate() throws IOException {
+        // given
+        final long testTime = Instant.now().toEpochMilli();
+
+        // when
+        final Response response = given(adminSpec).get("/currency-rates");
+
+        // then
+        System.out.println(response.asString());
+        final String lastUpdateValue = response.jsonPath().getString("last_update");
+        final long lastUpdateTime = Instant.parse(lastUpdateValue).toEpochMilli();
+        assertThat(testTime).isGreaterThan(lastUpdateTime);
```

Comment:
There is specialized `assertThat` method that accepts `Instant` and returns assertion providing time-specific methods. Please use that instead.